cli/spam.go

```diff
@@ -492,7 +492,11 @@ func startIssuer(cctx context.Context, issuer *txIssuer) {
 				if result.Success {
 					confirmedTxs++
 				} else {
-					utils.Outf("{{orange}}on-chain tx failure:{{/}} %s %t\n", string(result.Output), result.Success)
+					for i := 0; i < len(result.Outputs); i++ {
```

Comment:
You should respect the invariant you implicitly created before (where any revert just populates the first output of the first action) or ensure each action output has a value